### ISSUE TITLE
rss readmore link doesnt include full URL

### ISSUE THREAD

**Issue by nzyeehaa** [joomla/joomla-cms#23064]
### Steps to reproduce the issue
turn on RSS for category or blog
check feed 
readmore link unable to be clicked on feed reader/ RSS to email

### Expected result

click readmore link - go to page 

### Actual result

fail

### System information (as much as possible)

J3.9

### Additional comments
joomla-cms/components/com_content/views/category/view.feed.php

I replaced...
```
// URL link to article
$link = JRoute::_(ContentHelperRoute::getArticleRoute($item->slug, $item->catid, $item->language));
```
with 
```
// URL link to article
$link = JRoute::_(ContentHelperRoute::getArticleRoute($item->slug, $item->catid, $item->language), true, 1 );
```

**Comment by PhilETaylor** [comment]
ok I can replicate now - but I dont know the exact params you are setting, PR at #23098

**Comment by PhilETaylor** [comment]
Please close and test #23098

**Comment by nzyeehaa** [comment]
> I cannot replicate your issue
> 
> The XML Feed I get is
> 
> ```
> <?xml version="1.0" encoding="utf-8"?>
> 
> <rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
> 	<channel>
> 		<title>Category Blog</title>
> 		<description><![CDATA[]]></description>
> 		<link>http://0.0.0.0/index.php/category-blog</link>
> 		<lastBuildDate>Fri, 16 Nov 2018 12:28:07 +0000</lastBuildDate>
> 		<generator>Joomla! - Open Source Content Management</generator>
> 		<atom:link rel="self" type="application/rss+xml" href="http://0.0.0.0/index.php/category-blog?format=feed&amp;type=rss"/>
> 		<language>en-gb</language>
> 		<item>
> 			<title>Getting Started</title>
> 			<link>http://0.0.0.0/index.php</link>
> 			<guid isPermaLink="true">http://0.0.0.0/index.php</guid>
> 			<description><![CDATA[<p>It's easy to get started creating your website. Knowing some of the basics will help.</p>
> <h3>What is a Content Management System?</h3>
> <p>A content management system is software that allows you to create and manage webpages easily by separating the creation of your content from the mechanics required to present it on the web.</p>
> <p>In this site, the content is stored in a <em>database</em>. The look and feel are created by a <em>template</em>. Joomla! brings together the template and your content to create web pages.</p>
> <h3>Logging in</h3>
> <p>To login to your site use the user name and password that were created as part of the installation process. Once logged-in you will be able to create and edit articles and modify some settings.</p>
> <h3>Creating an article</h3>
> ]]></description>
> 			<category>test</category>
> 			<pubDate>Fri, 16 Nov 2018 12:21:31 +0000</pubDate>
> 		</item>
> 	</channel>
> </rss>
> ```
> 
> And all the urls in that are valid.

@PhilETaylor  Issue title hints to a READMORE link not including full url...
cant see readmore in your replication?

**Comment by nzyeehaa** [comment]
@PhilETaylor 
https://github.com/joomla/joomla-cms/issues/23064#issuecomment-439381866

default install
ADMIN->content->articles->options->integration
  ->include in feed -> intro text
  ->show readmore ->show
visit RSS

```
<?xml version="1.0" encoding="utf-8"?>
<!-- generator="Joomla! - Open Source Content Management" -->
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
	<channel>
		<title>News</title>
		<description><![CDATA[Joomla CMS]]></description>
		<link>http://localhost/j375/index.php/news</link>
		<lastBuildDate>Thu, 31 Oct 2019 04:22:39 +0000</lastBuildDate>
		<generator>Joomla! - Open Source Content Management</generator>
		<atom:link rel="self" type="application/rss+xml" href="http://localhost/j375/index.php/news?format=feed&amp;type=rss"/>
		<language>en-gb</language>
		<item>
			<title>Article 1 Title</title>
			<link>http://localhost/j375/index.php/news/3-article-1-title</link>
			<guid isPermaLink="true">http://localhost/j375/index.php/news/3-article-1-title</guid>
			<description><![CDATA[<p>Here is a news article.</p>
<p class="feed-readmore"><a target="_blank" href ="/j375/index.php/news/3-article-1-title">Read More ...</a></p>]]></description>
			<category>News</category>
			<pubDate>Tue, 13 Nov 2018 11:19:55 +0000</pubDate>
		</item>
	</channel>
</rss>
```

**Comment by nzyeehaa** [comment]
this still hasnt been fixed as of 3.9.22

ref: https://github.com/joomla/joomla-cms/pull/23098#issuecomment-548206553
